### DISCUSSION TITLE
AssetPrefix is ignored when page having getServersideProps is loaded

### DISCUSSION THREAD

**Discussion by seonghyeonkimm** [vercel/next.js#32849] [Help] [answered]
I used asseetPrefix in next.config.js and it works as expected when loading page which is prerendered (no getServersideProps). However, when loading page which is rendered on the server (having  getServersideProps), the page ignores assetPrefix and it loads static files from just **/_next/static/***.

I am figuring out why it happens. Is there anyone who had same experience with me?

**Comment by seonghyeonkimm** [comment, marked as answer]
I found the solution. I have run `next build` inside Docker.  And inside Dockerfile, I am using multi-stage docker build stage. However I am passing assetPrefix ENV to build stage not to runner stage

**Comment by Zwahxf** [comment]
how to do this ? for details. thanks
